File: introduction/ITP1_2_A-Small, Large, or Equal.java
import java.util.*;

public class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    String[] str = sc.nextLine().split(" ");
    int a = Integer.parseInt(str[0]);
    int b = Integer.parseInt(str[1]);
    if (a == b) {
      System.out.println("a == b");
    } else if (a > b) {
      System.out.println("a > b");
    } else {
      System.out.println("a < b");
    }
  }
}

File: introduction/ITP1_2_A-Small, Large, or Equal.scala
object Main {
  def main(args:Array[String]) = {
    val sc = new java.util.Scanner(System.in)
    val a = sc.nextInt
    val b = sc.nextInt
    if (a == b) {
      println("a == b")
    } else if (a > b) {
      println("a > b")
    } else {
      println("a < b")
    }
  }
}
